### PR TITLE
Rebuild for libabseil20240722_grpc165_libprotobuf5275

### PR DESCRIPTION
**Pull request by xhochy**
Fixes #404 

Checklist
* [x] Used a [personal fork of the feedstock to propose changes](https://conda-forge.org/docs/maintainer/updating_pkgs.html#forking-and-pull-requests)
* [x] Bumped the build number (if the version is unchanged)
* [ ] Reset the build number to `0` (if the version changed)
* [x] [Re-rendered]( https://conda-forge.org/docs/maintainer/updating_pkgs.html#rerendering-feedstocks ) with the latest `conda-smithy` (Use the phrase <code>@<space/>conda-forge-admin, please rerender</code> in a comment in this PR for automated rerendering)
* [ ] Ensured the license file is being packaged.

### CODE REVIEW

File: recipe/meta.yaml [hunk 1]
```diff
@@ -226,7 +226,7 @@ outputs:
         - ml_dtypes >={{ ml_dtypes_version }},<0.5
         - numpy
         - opt_einsum >=2.3.2
-        - protobuf >=3.20.3,<5,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
+        - protobuf >=3.20.3,<6,!=4.21.0,!=4.21.1,!=4.21.2,!=4.21.3,!=4.21.4,!=4.21.5
```
Comment (h-vetinari) [inline recipe/meta.yaml:229]: I think we can simplify that condition. The way our `protobuf` is built (at least before v5) is that it depends on `libprotobuf`, which would conflict with the run_export that's being updated through the migration here, so nothing before v5 would be co-installable with the `tensorflow` being built here anyway.
```suggestion
        # simplified compared to upstream due to constraints arising through libprotobuf
        - protobuf >=5.26,<6
```

**Review by h-vetinari — APPROVED** [review]
LGTM thanks! I guess we mainly need someone to build osx? Also pinging @hmaarrfk @ngam if you can help by any chance.

**Comment by xhochy** [comment]
It's a grey cold day here. I can do the rebuild and heat my house 🧣

**Comment by h-vetinari** [comment]
> It's a grey cold day here. I can do the rebuild and heat my house 🧣

That's amazing! 😊 

Could you just delete the second over-complicated protobuf-pin as well? I just commented in one place and thought the extension was obvious 😅

**Comment by xhochy** [comment]
> Could you just delete the second over-complicated protobuf-pin as well? I just commented in one place and thought the extension was obvious 😅

Sorry, was on mobile again 🙈

**Comment by xhochy** [comment]
[osx-64-logs.zip](https://github.com/user-attachments/files/17389288/osx-64-logs.zip)

**Comment by xhochy** [comment]
[osx-arm64-logs.zip](https://github.com/user-attachments/files/17389479/osx-arm64-logs.zip)

**Comment by h-vetinari** [comment]
Thank you very much! 🙏 ❤️

**Pull request merged**